.github/workflows/zapatos.yaml

```diff
@@ -0,0 +1,33 @@
+name: zapatos
+
+on: [push, pull_request]
+
+jobs:
+  ci:
+    runs-on: ubuntu-latest
+    container:
+      image: node:14.17.0-buster-slim
+    timeout-minutes: 3
+
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v2
+
+      - name: Cache dependencies
+        uses: actions/cache@v2
```

Comment:
I recommend to use the following, instead of `actions/cache` and `npm ci` step:

```
      - name: Install
        uses: bahmutov/npm-install@v1
```

Reply:
What is the advantage? I prefer to not use third-party actions unless they save me a lot of work. I also like the explicitness here. I know exactly how things are cached just by looking at this YAML file and if something doesn't work it's easier to debug why.

I'm happy to change it though if the consensus is that `bahmutov/npm-install` is better and widely used.

Reply:
Every action is a GitHub repository and open source, including the ones offered by GitHub. The line between first and third party is somewhat blurry as anyone is welcome to contribute to any action.

If you are worried about security you can specify the exact commit hash as an action version.

The reason I recommended the action is because it is well thought out action that covers many edge cases. I found that getting caching right wasn't always so easy. It uses the same caching action under the hood.

Reply:
I'm fine either way. I think all in all it's a minor detail that can be changed one way or the other later. @jawj what do you think?

Reply:
I agree. My comment was a recommendation based on experience. But it's definitely not a requirement. It should work as is.

Reply:
Thanks both. I think I'm more comfortable for now leaving in the original GitHub actions, but if we find those problematic at any point I'll bear the third-party recommendation in mind.